### PR TITLE
Add option to customize the instance logo

### PR DESCRIPTION
**Pull request by dennisreimann**
Custom logo for BTCPay instances, plus some view simplifications.

settings

login

hrf

login-spiral

### CODE REVIEW

File: BTCPayServer/Components/MainLogo/Default.cshtml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@model BTCPayServer.Components.MainLogo.MainLogoViewModel
+
+@if (!string.IsNullOrEmpty(Theme.LogoFileId))
+{
+    var logoSrc = await FileService.GetFileUrl(Context.Request.GetAbsoluteRootUri(), Theme.LogoFileId);
+    <img src="@logoSrc" alt="BTCPay Server" class="main-logo main-logo-custom @Model.CssClass" />
+}
+else
+{
+    var logoSrc = $"{ViewContext.HttpContext.Request.PathBase}/img/logo.svg";
```
Comment (NicolasDorier) [inline BTCPayServer/Components/MainLogo/Default.cshtml:16]: you don't need to do this, we have a tag helper `SVGUse` which already should insert the root path

Comment (dennisreimann) [reply BTCPayServer/Components/MainLogo/Default.cshtml]: Thanks for the pointer, done in [3426bc9](https://github.com/btcpayserver/btcpayserver/pull/4258/commits/3426bc983dafeb8a56d34bf376f20d90b8797f20).

**Review by pavlenex — APPROVED** [review]
tACK, this will make a lot of third-party hosts happy, great idea.

**Review by dstrukt — APPROVED** [review]
Thought my review went through .. marking for posterity.

**Pull request merged**